### DISCUSSION TITLE
WebSocket Implementation BlenderBot2

### DISCUSSION THREAD

**Discussion by Tacacs-1101** [facebookresearch/ParlAI#4345] [Q&A]
How to use the web socket implementation for multiple sessions in blenderbot2?

**Comment by klshuster** [comment]
Not sure what you mean by multiple sessions, but the browser chat and the terminal chat are both examples of using the websocket implementation and both ostensibly support multiple sessions (both in the sense of multiple chats and multiple people talking to the bot)

**Comment by Tacacs-1101** [comment]
I have tested the browser chat implementation, and it doesn't seem to be one to many. When other users try to talk, the context gets intermingled.

**Comment by Tacacs-1101** [comment]
If I have to deploy parlai in grpc server, which implementation can I replicate?
